cli/root42_subdomain_template.txt

```diff
@@ -0,0 +1,19 @@
+<subdomain>.<nest_user>.hackclub.dn42 {
```

Comment:
Might be worth setting the tls.ca option to `https://acme.burble.dn42/v1/dn42/acme/directory` for dn42 domains (may require adding the dn42 CA cert to nest's CA cert list so the acme server can be connected to) so that automatically issuing certs for dn42 domains would work (https://caddyserver.com/docs/caddyfile/directives/tls#ca)

Reply:
I could do that. Is that the only field I would need to set?

Reply:
I think so, you'd probably just have to additionally make sure that resolving dn42 domains on nest works

Reply:
I think I will have to add this at a later point